### PR TITLE
Exlude /tests for dist package

### PR DESCRIPTION
**Pull request by LeoColomb**
Follow guidelines to clean up dist packages
http://www.sitepoint.com/mastering-composer-tips-tricks/#making-your-composer-project-lighter

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Should be alphabetically sorted and without a slash, then :+1: from me.

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .gitattributes export-ignore
 .gitignore export-ignore
 .travis.yml export-ignore
+/tests
```
Comment (GrahamCampbell) [inline .gitattributes]: You forgot to say export-ignore...

Comment (LeoColomb) [inline .gitattributes]: Oh my god...

**Comment by LeoColomb** [comment]
Fixed.

> without a slash

Why?

**Comment by GrahamCampbell** [comment]
> Why?

For consistancy.

**Comment by GrahamCampbell** [comment]
The albabetical sort is wrong.

**Comment by GrahamCampbell** [comment]
dots come first, then abcd.

**Comment by GrahamCampbell** [comment]
Correct order...

```
.gitattributes export-ignore
.gitignore export-ignore
.travis.yml export-ignore
phpunit.xml.dist export-ignore
tests export-ignore
```

**Comment by LeoColomb** [comment]
Oh, sorry for order, I had put this (undotted alphabetical) order: https://github.com/thephpleague/skeleton/blob/c5f84b07b8dd331c70d3c5b6bc403084ab0be0c6/

But thanks for your feedback!

**Pull request merged**